webhook/init.go

```diff
@@ -153,6 +153,15 @@ func (w *WebHook) GetRequest() (req *http.Request) {
 
 		w.SetCustomHeaders(req)
 		SetGitHubHeaders(req, payload, *w.GetSecret())
+	} else if w.GetType() == "gitlab" {
```

Comment:
Sorry for being fussy, but could we make this a `switch` please?
So something like
```go
	switch w.GetType() {
	case "github":
	case "gitlab":
	}
```

Reply:
Yes I can Change this later the day.